src/api/0x/0x.spec.ts

```diff
@@ -100,3 +100,126 @@ test('Invalid: Get Price Quote', async () => {
     expect(fetchMock).toHaveBeenCalledTimes(1)
   }
 })
+// normally this won't happen - but just in case we get 400 with no
+// standard error response body
+test('Invalid: Get Price Quote - 400', async () => {
+  fetchMock.mockResponseOnce(JSON.stringify({}), { status: 400 })
+  try {
```

Comment:
Same comment here as I made in the other PR https://github.com/cowprotocol/cow-sdk/pull/47

Can you try to not use `try`?

Reply:
sure - just a heads up this is the way it was done in cow.spec so followed that pattern

Reply:
I know, I figured out about this pattern afterwards, and the code already in place hasn't been refactored.